setup.py

```diff
@@ -37,7 +37,7 @@
         "packaging",
         "scipy",
         "sentencepiece",
-        "torch>=1.7,<=1.11",
```

Comment:
Wouldn't this cause an error ? 1. nothing after

Reply:
The `7` is replaced by a `9` to try and synchronize the `requirements.txt` and `setup.py` to have the same minimum version of torch. I'm not totally sure why it was different to begin with, if you know it was intentional we can leave it as `1.7`

Reply:
dunno ...